packages/strapi-design-system/src/ToggleCheckbox/ToggleCheckbox.js

```diff
@@ -76,35 +75,23 @@ export const ToggleCheckbox = React.forwardRef(
           borderWidth="1px"
           borderColor="neutral200"
         >
-          <ValueBox
-            paddingLeft={7}
-            paddingRight={7}
-            aria-hidden={true}
-            checked={checked === null ? false : !checked}
-            disabled={disabled}
-          >
+          <ValueBox paddingLeft={7} paddingRight={7} aria-hidden={true} checked={checked === null ? false : !checked}>
             <Typography
               variant="pi"
               fontWeight="bold"
               textTransform="uppercase"
-              textColor={disabled ? 'neutral600' : offCheckboxLabelColor}
+              textColor={disabled ? 'neutral500' : offCheckboxLabelColor}
             >
               {offLabel}
             </Typography>
           </ValueBox>
 
-          <ValueBox
-            paddingLeft={7}
-            paddingRight={7}
-            aria-hidden={true}
-            checked={checked === null ? false : checked}
-            disabled={disabled}
-          >
+          <ValueBox paddingLeft={7} paddingRight={7} aria-hidden={true} checked={checked === null ? false : checked}>
             <Typography
               variant="pi"
               fontWeight="bold"
               textTransform="uppercase"
-              textColor={disabled ? 'neutral600' : onCheckboxLabelColor}
+              textColor={disabled ? 'neutral500' : onCheckboxLabelColor}
```

Comment:
Same here about the color contrast issue

Reply:
Addressed in 8cbac4fbe4f47ee42c1c2bb32d0856b1ee6dced1. Discussion continues above.